src/main/java/se/bjurr/violations/lib/example/MyClass.java

```diff
@@ -30,6 +31,6 @@ void npe2(String a, String b) {
```

Comment:
**Reporter**: Findbugs
**Rule**: NP_ALWAYS_NULL
**Severity**: ERROR
**File**: se/bjurr/violations/lib/example/MyClass.java L13
**Source**: se.bjurr.violations.lib.example.MyClass

Null pointer dereference&#10;&#10;&#10;&lt;p&gt; A null pointer is dereferenced here.&amp;nbsp; This will lead to a&#10;&lt;code&gt;NullPointerException&lt;/code&gt; when the code is executed.&lt;/p&gt;&#10;


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1442528971>*